fixtures/cameo/auro-spot-400.json

```
@@ -23,6 +23,9 @@
     "dimensions": [405, 565, 285],
     "weight": 15.6,
     "power": 290,
+    "powerConnectors": {
+      "powerCON": "input and ouput"
```

```suggestion
      "powerCON": "input and output"
```

Noticed by the [`fixtures-valid` test](https://github.com/OpenLightingProject/open-fixture-library/actions/runs/5391771774/jobs/9789135981?pr=3350).